sycl/test/basic_tests/accessor/Inputs/device_accessor.cpp

```diff
@@ -0,0 +1,319 @@
+//==---------- device_accessor.cpp - SYCL accessor basic test --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include <CL/sycl.hpp>
+#include <cassert>
+
+namespace sycl {
+using namespace cl::sycl;
+}
+
```

Comment:
```suggestion
```

This is not necessary. Please, remove from other tests too.

Reply:
done